md/bdm-access-control.md

```diff
@@ -5,7 +5,7 @@
 :::
 
 ## Overview
-This page explains how to protect the access to your BDM through REST API calls via a white list mechanism.
+This page explains how to protect the access to your BDM through REST API calls via a white list mechanism. It does not apply on DAO API.
```

Comment:
"This protection does not secure access to BDM through [DAO](https://en.wikipedia.org/wiki/Data_access_object)."